src/components/ConversionFileErrorModal/index.tsx

```diff
@@ -33,7 +32,7 @@ const ConversionFileErrorModal: React.FC<ConversionFileErrorModalProps> = ({
             onCancel={closeModal}
         >
             <div>
-                <div className={styles.redText}>
+                <div className={styles.redWarningText}>
                     {" "}
```

Comment:
I know this isn't part of the code changes, but why are these spaces here?

Reply:
i think those are artifacts of using prettier to format the document